packages/decentraland-loader/lifecycle/manager.ts

```diff
@@ -122,10 +124,11 @@ export async function initParcelSceneWorker() {
   const state = store.getState()
 
   const fullRootUrl = getResourcesURL('.')
+  const localServer = resolveUrl(`${location.protocol}//${location.hostname}:${8080}`, '/local-ipfs')
 
   server.notify('Lifecycle.initialize', {
-    contentServer: getFetchContentServer(state),
-    catalystServer: getCatalystServer(state),
+    contentServer: ENABLE_TEST_SCENES ? localServer : getFetchContentServer(state),
+    catalystServer: ENABLE_TEST_SCENES ? localServer : getCatalystServer(state),
```

Comment:
Does it work without this? I don't remember what was `catalystServer` used for?

Reply:
It does not 😔
It won't load the test-scenes without that